CMakeLists.txt

```diff
@@ -98,6 +98,17 @@ if(HAIKU)
   set(ASAN OFF)
 endif()
 
+if(AMIGA)
+  set(ASAN OFF)
+  set(UBSAN OFF)
+  set(NONET ON)
+  set(USE_SDL1 ON)
+  find_package(Freetype REQUIRED)
+  find_package(ZLIB REQUIRED)
+  set(CMAKE_CXX_FLAGS "${CMAKE_C_FLAGS} -fexceptions")

```

Comment:
Did you mean `${CMAKE_CXX_FLAGS}`? Also please add a comment explaining why we're doing this for AMIGA

Reply:
By default, exceptions support are disabled in the Amiga toolchain as it is slow, we have exceptions in our net code so it need to be enabled for things to compile.
Fixed and comment added.